### PR TITLE
VFIO Measurements and Packet Size Loop Variable

### LINKED ISSUE #44: autotest: passthrough test
1. measure passed through E810. 
    - bind dpdk driver on host
    - start qemu with device like [1]
2. measure passed thorough E810 with vmux mediation
    - same as 1., but start vmux server `sudo ./build/vmux` before qemu
    - use qemu with libvfio-user (built here `nix build .#qemu` [2])
    - start qemu with other flag [3]
    
[1] https://github.com/vmuxIO/vmuxIO/blob/e29f615030e3cd68d7e533e6f84faca78c90c4c8/justfile#L43
[2] https://github.com/vmuxIO/vmuxIO/blob/e29f615030e3cd68d7e533e6f84faca78c90c4c8/justfile#L130
[3] https://github.com/vmuxIO/vmuxIO/blob/e29f615030e3cd68d7e533e6f84faca78c90c4c8/justfile#L57

### LINKED ISSUE #52: Autotest Packet Size Loop Variable
It would be helpful if the packet size was another loop variable in autotest.

### PR DESCRIPTION
**Pull request by gierens**
Alter autotest for VFIO measurements and make packet size a loop variable for load latency tests.

Resolves #44 and resolves #52

### CODE REVIEW

**Comment by pogobanane** [comment]
guest-image fails to build in CI. Please fix. Does it build locally for you?

File: src/vfio-consumer.cpp [hunk 1]
```diff
@@ -128,7 +128,11 @@ int VfioConsumer::init() {
   this->dma_map = dma_map;
 
   /* Get a file descriptor for the device */
+<<<<<<< HEAD
+  device = ioctl(group, VFIO_GROUP_GET_DEVICE_FD, "0000:51:00.0");
+=======
   device = ioctl(group, VFIO_GROUP_GET_DEVICE_FD, device_name.c_str());
+>>>>>>> main
```
Comment (pogobanane) [inline src/vfio-consumer.cpp:135]: hust hust

File: nix/guest-config.nix [hunk 2]
```diff
@@ -75,64 +75,64 @@
...
-  #         '';
-  #       } {
-  #         name = "remove-useless-stuff";
-  #         patch = null;
-  #         extraConfig = ''
-  #           CONFIG_USB n
-  #           CONFIG_WLAN n
-  #         '';
-  #       } ];
+        src = fetchurl {
```
Comment (RaitoBezarius) [inline nix/guest-config.nix:85]: this should use `fetchFromGitHub` rather than relying on the URL

File: src/vfio-consumer.cpp (hunk 1 above)
Comment (gierens) [reply src/vfio-consumer.cpp]: my bad :D ... fixed in ca4b0e8 ;)

**Pull request merged**